vitrina/orgs/views.py

```diff
@@ -180,6 +183,14 @@ def form_valid(self, form):
             )
             messages.info(self.request, _("Naudotojui išsiųstas laiškas dėl registracijos"))
         self.object.save()
+
+        if self.object.has_api_access:
+            ApiKey.objects.create(
+                api_key=str(uuid.uuid4()),
```

Comment:
Previously for secret keys uuid4 was used, but it is not really meant for this, So I suggest to use more secure [secrets.token_urlsafe()](https://docs.python.org/3/library/secrets.html#secrets.token_urlsafe) function.

```suggestion
                api_key=secrets.token_urlsafe(),
```